From 90510b47f88da5463fc64abb199669b6e2f45af3 Mon Sep 17 00:00:00 2001
From: tgeisler <tgeisler@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 31 May 2012 16:26:39 +0000
Subject: [PATCH 0204/1000] MANTIS0023436: Mcontrol - W7 dialog pops up if the
 string "update" is found in a name. Rename mUpdate.exe to mUpdat.exe

git-svn-id: svn://localhost/SelfServiceCommon/trunk@338 90b65887-3827-0410-9a23-83215b262276
---
 .../Massai/cpp/Controller/inc/mcontrol_def.h           | 10 +++++++---
 .../Massai/cpp/Controller/src/serverwin.cpp            |  2 +-
 2 files changed, 8 insertions(+), 4 deletions(-)

diff --git a/SelfServiceCommon/Massai/cpp/Controller/inc/mcontrol_def.h b/SelfServiceCommon/Massai/cpp/Controller/inc/mcontrol_def.h
index f141a45f..217c3fb9 100644
--- a/SelfServiceCommon/Massai/cpp/Controller/inc/mcontrol_def.h
+++ b/SelfServiceCommon/Massai/cpp/Controller/inc/mcontrol_def.h
@@ -9,15 +9,19 @@
  *
  ************************************************************************/
 
-#define FILE_VER_BIN     1,0,2,3
-#define FILE_VER_STR     "FileVersion", "1.0.2.3\0"
+#define FILE_VER_BIN     1,0,2,4
+#define FILE_VER_STR     "FileVersion", "1.0.2.4\0"
 
 /*! \page mControl.exe - platform startup, starts mudpate.exe, mstarter.exe; servant for systemControl interface
  *
- *  @version 1.0.2.3
+ *  @version 1.0.2.4
  *
  *  \section change1 Change history
  *
+ *  \date 31.05.2012 \li V1.0.2.4
+ *                   \li bugfix - 0023436: Mcontrol - W7 dialog pops up if the string "update" is found in a name.
+ *                   \li Rename mUpdate.exe to mUpdat.exe
+ *
  *  \date 07.05.2012 \li V1.0.2.3
  *                   \li bugfix - Mantis 0023196: MControl needs ADMINISTRATOR rights to start
  *
diff --git a/SelfServiceCommon/Massai/cpp/Controller/src/serverwin.cpp b/SelfServiceCommon/Massai/cpp/Controller/src/serverwin.cpp
index 5721875a..9fa98db3 100644
--- a/SelfServiceCommon/Massai/cpp/Controller/src/serverwin.cpp
+++ b/SelfServiceCommon/Massai/cpp/Controller/src/serverwin.cpp
@@ -45,7 +45,7 @@
 
 #define SEM_NAME       "mPlatformStartup"
 #define STARTER_NAME   "mstarter.exe"
-#define UPDATE_NAME    "mupdate.exe"
+#define UPDATE_NAME    "mUpdat.exe"
 #define REBOOT_KIOSK   123
 
 #define START_DELAY_TIMER      1
-- 
2.41.0.windows.1

